src/main/java/org/casbin/jcasbin/util/LRUCache.java

```diff
@@ -0,0 +1,105 @@
+// Copyright 2022 The casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.casbin.jcasbin.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Yixiang Zhao (@seriouszyx)
+ **/
+public class LRUCache<K, V> {
```

Comment:
Consider using LinkedHashMap directly to implement LRU cache

Reply:
@seriouszyx